packages/client/src/__tests__/integration/happy/composites-mongo/findMany/required.ts

```diff
@@ -90,4 +90,38 @@ describeIf(!process.env.TEST_SKIP_MONGODB)('findMany > required', () => {
       ]
     `)
   })
+
+  /**
+   * Order by
+   */
+  test('orderBy', async () => {
+    const comment = await prisma.commentRequiredProp.findMany({
+      where: { id },
+      orderBy: {
+        content: {
+          upvotes: {
+            _count: 'desc',
+          },
+        },
+      },
+    })
+
+    expect(comment).toMatchInlineSnapshot(`
+      Array [
+        Object {
+          content: Object {
+            text: Hello World,
+            upvotes: Array [
+              Object {
+                userId: 10,
+                vote: true,
+              },
+            ],
+          },
+          country: France,
+          id: 3aaaaaaaaaaaaaaaaaaaaaaa,
+        },
+      ]
+    `)
+  })
```

Comment:
Since there is only one item in upvotes it's hard to say if the orderBy upvotes works here, could you add one more item?

Reply:
I think it applies to many tests here

Reply:
Fixed